src/components/Modal/ModalBackground.jsx

```diff
@@ -2,11 +2,18 @@ import useScrollDisable from "../../hooks/useScrollDisable";
 import useBoundStore from "../../store/client/useBoundStore";
 import PropTypes from "prop-types";
 
-const ModalBackground = ({ modalType, children }) => {
+const ModalBackground = ({ isClear, modalType, children, isDataFetching }) => {
```

Comment:
```suggestion
const ModalBackground = ({ isClear, modalType, children, isFetchingData }) => {
```
단어 순서에 따라 의미가 조금 달라지다보니 `isFetchingData`으로 변수명 수정해보면 어떨까요?
'데이터 패칭 중' 상태의 의미를 담으려면 `isFetchingData` 등으로 고려해볼 수 있을 것 같습니다.